view/code/index.ts

```diff
@@ -104,6 +110,20 @@ function listenChanges(input: HTMLInputElement): void {
   })
   input.addEventListener('blur', () => {
     locked.set(input, false)
+
+    let value = input.value.trim()
+
+    let currentFormat = outputFormat.get()
+    let newFormat = toOutputFormat(value)
+
+    if (
+      !(currentFormat === 'figmaP3' && isHex8(value)) &&
```

Comment:
Figma always (including P3 mode) is using hex 6.

On Figma P3 mode we should just keep this mode and do not change it.